_security-analytics/api-tools/mappings-api.md

```diff
@@ -11,6 +11,17 @@ The following APIs can be used for a number of tasks related to mappings, from c
 
 ## Get Mappings View
 
+Returns a view of the fields contained in an index used as a log source.
```

Comment:
Please rephrase as a complete sentence naming the noun (what is used to do this?).